app/views/users/show.html.erb

```diff
@@ -1,2 +1,35 @@
-<h1>User Profile</h1>
-<!-- Placeholder for displaying single user profile -->
\ No newline at end of file
+<h2>User Profile</h2>
+<main>
+  <div class="user-container">
+    <div class="profile-pic">
+      <img src="<%= @user.photo %>" alt="profile-picture">
+    </div>
+    <div class="user-info">
+      <h2><%= @user.name %></h2>
+      <p class="posts-counter">Number of posts: <%= @user.posts_counter || 0 %></p>
+    </div>
+  </div>
+  <article class="user-bio">
+    <h5>Bio</h5>
+    <div class="bio"><%= @user.bio %></div>
+  </article>
+  <article>
+    <% @posts.each do |post| %>
+      <%= link_to post.title, user_post_path(@user, post) %>
+      <div class="post-container">
+        <h5 class="post-title"><%= post.title %></h5>
+        <% if post.text.present? %>
+          <p><%= post.text[0..100] %>...</p>
+        <% else %>
+         <p>No text available...</p>
+        <% end %>
+      </div>
+      <div class="comment-likes-div">
+        <p>Comment: <%= post.comments_counter || 0 %>, Likes: <%= post.likes_counter || 0 %></p>
+      </div>
+    <% end %>
+  </article>
+  <div>
+    <%= link_to "See all posts", user_posts_path(@user) %>
+  </div>
+</main>
```

Comment:
| @WilhelmK109 User Show Page | Microverse User Show Page |
|-|-|
| ![image](https://github.com/WilhelmK109/BlogApp/assets/93039359/59f46b8b-788f-45da-963b-61561ca18098) | ![image](https://github.com/WilhelmK109/BlogApp/assets/93039359/76f1f2d2-6564-41c1-b6a8-a8c28bbde160) |

- [ ] Great work so far but here we need minor changes 😊 Kindly expand your data on the whole screen as you can see in the given design 👀 that the Name is on the left side and Posts Counter is on the right side 🤔 to organize the user information well 👍 Kindly make this change 📝
- [ ] It would be nice if you make the likes counter and comments counter on the right position of the posts as you can see in the given design to make your app look identical to the given design to make your app professional 👍
- [ ] It would be nice if you made the `See all posts` button in the center and make it look identical to the button in the given design to make your design look professional 👍